source/collection-management/tests/timeseries-collection.yml

```diff
@@ -103,13 +113,17 @@ tests:
           - commandStartedEvent:
               command:
                 create: *collection0Name
+                expireAfterSeconds: 604800
                 timeseries: *timeseries0
               databaseName: *database0Name
           - commandStartedEvent:
               command:
                 insert: *collection0Name
                 documents: *docs
-    outcome:
-    - collectionName: *collection0Name
-      databaseName: *database0Name
-      documents: *docs
+          - commandStartedEvent:
+              command:
+                find: *collection0Name
+                filter: {}
+                sort: { time: 1 }
+              commandName: find
```

Comment:
I think we covered this in a previous PR, but `commandName` really servers no purpose in `expectEvents` and I regret blindly keeping it around from the legacy APM tests. The only necessary use of `commandName` is with `runCommand` operations.

I'd suggest removing this, as you don't have it for the other commands in this test.

Reply:
Ah yep; since I'm matching the `command` already, removed `commandName` from this assertion.